app/organizations.js

```diff
@@ -1,33 +0,0 @@
-export default [
-  {
-    name: 'Discourse',
-    query: { q: 'is:open is:issue user:discourse' },
-  }, {
-    name: 'Wikimedia',
-    query: { q: 'is:open is:issue user:wikimedia' },
-  }, {
-    name: 'openSUSE',
-    query: { q: 'is:open is:issue user:opensuse' },
-  }, {
-    name: 'Elm',
-    query: {q: 'is:open is:issue user:elm-lang'},
-  }, {
-    name: 'coala',
-    query: { q: 'is:open is:issue user:coala label:difficulty/newcomer no:assignee' },
```

Comment:
This functionality of specific labels is being removed in this patch?

I see pull requests in the results, so `is:issue` is also not being implemented.

Also issues with an assignee are in the results, so `no:assignee` has been lost also.

Reply:
Wikidata has nothing to do with this.  I will review https://github.com/coala/git-task-list/pull/37 later today, which will help explain why, but for now, just accept the following..

`is:issue` needs to be implemented in this PR, applying to all orgs.  Please remove everything regarding pull requests out of this PR.  You will need to re-add it in phase 2, so create a backup of this branch for later.  issues and PRs are different types of tasks, and shouldnt be done with the same UI.  We need to do design for PRs later -- not enough time to do it in this PR.

`no:assignee` needs to be implemented in this PR.  It was previously only for coala org, but you should re-implement it for every org, as it is extremely common that orgs do not want to advertise tasks which are already assigned to someone.

`label:difficulty/newcomer` means you need to allow the configuration of each organisation to include a label which will be used to get issues of.  When this label has been given by the org, issues without that label should not appear.  In the future, orgs will want to have multiple labels, OR'd together.